Yoast/yoast-components
# Add recommended replacement variables

## Summary

This PR can be summarized in the following changelog entry:

* Adds the option to the snippet editor to start with recommended replacement variables only.

## Relevant technical choices:

* If the `searchValue` is empty and there are `recommendedReplacementVariables` show the `recommendedReplacementVariables` instead of the `replacementVariables`.

## Test instructions

This PR can be tested by following these steps:

* In the standalone.
* When typing `%` you should see only `title` and `post_type`.
<img width="412" alt="recommended_replacement_variables" src="https://user-images.githubusercontent.com/35524806/41402167-e06c9584-6fc1-11e8-9e08-aded14427eb1.png">

* When continuing to an `s` (for example). You should see the the `snippet` and `snippet_manual` too.

Fixes https://github.com/Yoast/wordpress-seo/issues/9888
